influxql/select_test.go

```diff
@@ -2645,6 +2645,104 @@ func TestSelect_Elapsed_Boolean(t *testing.T) {
 	}
 }
 
+func CheckPoints(t *testing.T, ic *IteratorCreator, query string, expected [][]influxql.Point) {
```

Comment:
Yup, I just thought that rather than duplicating the same code loads of times it would be best practice to write it once and wrap it in a function - but I'm happy to remove the function and get copy-pasting if that is the preferred style!